dist/index.html

```diff
@@ -4,31 +4,92 @@
+        <button type="button" class="nav-home">
+            <i class="fas fa-home"></i> Home
+        </button>
+        <button type="button" class="nav-projects">
+            <i class="fas fa-project-diagram"></i> Projects
+        </button>
+        <button type="button" class="nav-about">
+            <i class="fas fa-info-circle"></i> About Me
+        </button>
+        <button type="button" class="nav-contact">
+            <i class="fas fa-comment-dots"></i> Contact</button>
+        <div class="lower-nav">Custom Designs · Marketing · Branding · And More</div>
       </div>
-      <span class="perks">
-        Custom Designs · Marketing · Branding · And More
-      </span>
-      <div class="whois">
-        <h2> WHO IS SEEK-A-DESIGN?</h2>
-        </br>
-        <p>Seek-A-Design was a long-time dream of owner and designer, Lyandra Sica (seek•uh). Using her last name, she developed the play-on-words into her business which has sprouted vivdly within the last few years. Within those years, her hard work and determination has allowed her dream to turn into the reality of being a go-to designer and printing expert.</p>
+
+      <div class="whoisseeking-container">
+        <div class="whois">
+          <h2> WHO IS SEEK-A-DESIGN?</h2>
+          <hr>
+          <div class="whois-text">
+            <p>
+              Seek-A-Design was a long-time dream of owner and designer, Lyandra Sica (seek•uh). Using her last name, she developed the play-on-words into her business which has sprouted vivdly within the last few years. Within those years, her hard work and determination has allowed her dream to turn into the reality of being a go-to designer and printing expert.
+            </p>
+
+            <p>
+              Ever since Lyandra was four years old, she knew she wanted to be a creator. She accomplished numerous art courses in high school and graduated from New York Institute of Technology in Manhattan with a Bachelor's in Graphic Design. She understood and learned the field has many aspects beneath it such as advertising, packaging, editorial, and book design, but none caught her attention. What truly caught Lyandra's eye was the second half of the design operation, print and production. She finds it to be the most important piece of a finalized creation, because without print, a design will only be just a design.
+            </p>
+
+            <p>
+              No matter where she has been, she's produced superior work because she dedicates making sure each and every piece stands to return becuase of the attention to detail and the stress-free process that she prides herself on.
+            </p>
+          </div>
+        </div>
+
+        <div class="seeking">
+          <h2>SEEKING-A-REVIEW?</h2>
+          <hr>
+          <div class="seeking-text">
+            <ul>
+              <li>
+                “We had the most amazing wedding invitations created by Seek-A-Design!
```

Comment:
Love how you chose to wrap the text here! ✅

*Can we just update it a bit to make it less than 80 characters wide on each line?*